viewer/src/public/createRevealManager.ts

```diff
@@ -48,7 +48,8 @@ export function createRevealManager<T>(
   client: ModelDataClient<T>,
   revealOptions: RevealOptions = {}
 ): RevealManager<T> {
-  initMetrics(revealOptions.logMetrics !== false, project, {
+  const applicationId = client.getApplicationIdentifier();
+  initMetrics(revealOptions.logMetrics !== false, project, applicationId, {
     moduleName: 'createRevealManager',
     methodName: 'createRevealManager',
     constructorOptions: omit(revealOptions, ['internal'])
```

Comment:
I guess we can remove omitting here since you disabled metrics for examples. And just pass options without omitting.